prototypes_exercise/prototypes.js

```diff
@@ -0,0 +1,78 @@
+function Person(
+	firstName, lastName, favoriteColor, favoriteNumber, favoriteFoods =[]
+	)
+{	//"Elie", "Schoppik", "purple", 34
+	this.firstName = firstName;
+	this.lastName = lastName;
+	this.favoriteColor = favoriteColor;
+	this.favoriteNumber = favoriteNumber;
+	this.favoriteFoods = favoriteFoods;
+	this.fullName = function(){ return `${this.firstName} ${this.lastName}`;};
```

Comment:
Add this to the prototype instead of the object.  That way you reduce your memory usage.